spartan/devnet-smoke-tests/install_aztec.sh

```diff
@@ -0,0 +1,3 @@
+# Install aztec and set version
+bash -i <(curl -s https://install.aztec.network)
+aztec-up -v 0.81.0
```

Comment:
There's hardcoded references to this version in the job description, perhap's it'd be good to have these e.g. in an envvar to avoid them diverging.